install/ui/util/make-css.sh

```diff
@@ -22,5 +22,5 @@ set -o errexit
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-py3-lesscpy -x $DIR/../less/ipa.less > $DIR/../css/ipa.css
+python3 -m lesscpy -x $DIR/../less/ipa.less > $DIR/../css/ipa.css
```

Comment:
A small comment here: on RHEL 8.0 beta we have platform python and it is set as PYTHON:
```
export PYTHON=%{__python3}
```
Since build environment in RHEL 8.0 beta has no python3 but rather uses `platform-python`, this change will break build there. Perhaps, it is better to check if PYTHON is set and use that one instead?

```
${PYTHON:-python3} -m lesscpy -x ...
```

Reply:
Thanks, I forgot about platform-python. In the end it was simpler to replace the shell script with a one-liner in make: ``$(PYTHON) -m lesscpy -x ../less/ipa.less > $@``.